.carve_ignore

```diff
@@ -0,0 +1,6 @@
+; REVIEW Can some of these items be removed (later)?
+athens.db/help-url
+athens.db/ego-url
+athens.views/home-panel
+athens.core/init
+user/str-kw-mappings
```

Comment:
- URLs could still be used if we want to examine other DBs. especially `help-url`
- `home-panel` can be removed
- `str-kw-mappings` will be used for JSON import I believe
- not sure why `init` is not being used